physicsnemo/models/domino/model.py

```diff
@@ -211,49 +261,66 @@ def __init__(self, input_filters, model_parameters):
         self.activation = F.relu
         self.batch_norm = False
```

Comment:
batch norm is set to false here, and is not configurable at the user level.  Further, batch_norm doesn't make sense in this model since the batch_size is always 1.  

If no normalization will be used, we should remove calls in the forward bass and not construct the layers, it's just extra parameters.  If normalization will be used, LayerNorm or GroupNorm would make more sense in this scenario.

Reply:
Yup, I buy this argument - good call. Removing.

Reply:
Fixed in 46a09ddbb43fce1a4db1044b1f428dd97a3d8099